docs/builds/guides/development/custom-builds.md

```diff
@@ -106,12 +106,12 @@ module.exports = {
 		'../relative/path/to/some/othercustomplugin'
 	],
 
-	// UI language. Language codes follow the https://en.wikipedia.org/wiki/ISO_639-1 format.
-	language: 'en',
-
 	// Editor configuration.
 	config: {
-		toolbar: [ 'headings', 'bold', 'italic', 'custombutton' ]
+		toolbar: [ 'headings', 'bold', 'italic', 'custombutton' ],
+
+		// UI language. Language codes follow the https://en.wikipedia.org/wiki/ISO_639-1 format.
+		lang: 'en'
```

Comment:
One problem I see here is that the editor's config option is called `lang` while the Webpack plugin's `language`. This will lead people misconfiguring one or the other.

Since `additionalLangs` and `langs` sound at least bad, I think it'll be better if we used the full `language` word in all cases. So we need to change it in the editor's code.